src/main/java/org/concordion/api/AbstractCommand.java

```diff
@@ -25,4 +25,9 @@ public boolean isExample() {
 
     public void executeAsExample(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
     }
+
+    public boolean alwaysRunEvenIfNoNonExampleChildren() {
+        return false;
+    }
+

```

Comment:
This is one of the breaking API changes. Basically, our code to decide to execute an example was failing on table rows - because these have no children. This was my workaround, but open to new ideas.